src/commands/dependency/install.ts

```diff
@@ -43,10 +47,17 @@ export default class Install extends SfpCommand {
             ProjectConfig.getSFDXProjectConfig(null),
             this.flags.installationkeys
         );
-        let externalPackage2s = await externalPackageResolver.resolveExternalPackage2DependenciesToVersions();
+
+        let packages = null;
+        if (this.flags.releasefile) {
```

Comment:
the flag name is "releaseconfig", however is read as "filename"

Reply:
indeed, missed it when I unified option name with the rest of sfp 😬